src/base/bittorrent/session.cpp

```diff
@@ -1165,22 +1164,24 @@ void Session::configure(lt::settings_pack &settingsPack)
 
 #ifdef Q_OS_WIN
         // On Vista+ versions and after Qt 5.5 QNetworkInterface::name() returns
-        // the interface's Luid and not the GUID.
+        // the interface's LUID and not the GUID.
         // Libtorrent expects GUIDs for the 'outgoing_interfaces' setting.
-        if (!networkInterface().isEmpty()) {
-            const QString guid = convertIfaceNameToGuid(networkInterface());
+        const QString netInterface = networkInterface();
+        if (!netInterface.isEmpty()) {
+            const QString guid = convertIfaceNameToGuid(netInterface);
             if (!guid.isEmpty()) {
                 settingsPack.set_str(lt::settings_pack::outgoing_interfaces, guid.toStdString());
             }
             else {
                 settingsPack.set_str(lt::settings_pack::outgoing_interfaces, chosenIP.toStdString());
-                LogMsg(tr("Could not get GUID of configured network interface. Binding to IP %1").arg(chosenIP)
+                LogMsg(tr("Could not get GUID of configured network interface. Binding to IP: \"%1\"").arg(chosenIP)
```

Comment:
I see no reason to enclose IP and port in quotes in all such messages.

Reply:
done.